librosa/filters.py

```diff
@@ -571,6 +571,10 @@ def constant_q(sr, fmin=None, n_bins=84, bins_per_octave=12, tuning=0.0,
                                   'filter_scale', filter_scale,
                                   '0.4.2', '0.5.0')
 
+    # If filter_scale is None, auto-tune it based on frequency resolution
+    if filter_scale is None:
+        filter_scale = (2.0**(1./bins_per_octave) - 1.0)/(2.0**(1./12) - 1.0)
```

Comment:
Yeah. This might actually just take a small refactor.  Ideally, it should be isolated to cqt_filters, but the value is actually needed in a few places.